packages/xo-web/src/common/intl/messages.js

```diff
@@ -1023,6 +1023,7 @@ const messages = {
   hostSupportEnabled: 'XCP-ng Pro Support enabled on this host',
   noMoreMaintained: 'This host version is no longer maintained',
   pubKeyTooShort: 'TLS key is too small to update host to XCP-ng 8.3',
+  pubKeyTooShortLink: 'Follow this link for more details',
```

Comment:
The message ID should represent the meaning of the message so that we can reuse it as much as possible. The message doesn't say anything about a public key so the ID could be instead: `'clickLinkForDetails'`.